users/tests/conftest.py

```diff
@@ -305,14 +310,15 @@ def start_oidc_authorize(
         order=1,
     )
 
-    redirect_uris = ['https://example.com/callback']
-    oidc_client = oidcclient_factory(redirect_uris=redirect_uris)
+    redirect_uri = 'https://example.com/callback'
+    oidc_client_kwargs = {"redirect_uris": [redirect_uri]} | (oidc_client_kwargs or {})
```

Comment:
Or operator support in dicts is a Python 3.9 feature, I think. Could be nice to keep Tunnistamo compatible with (at least) all the supported Python 3 versions.

Even if the or would work with older Python versions I think this could be clearer as something like:
```python
    if oidc_client_kwargs is None:
        oidc_client_kwargs = {}
    oidc_client_kwargs.setdefault('redirect_uris', [redirect_uri])
```